### PR TITLE
Lesson 4

### PR DESCRIPTION
**Pull request by SobolevD51**
есть ошибки, нужна помощь))

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  left: 416px;
+  top: 419px;
+}
+.vector2 {
+  position: absolute;
+  left: 450px;
+  top: 279px;
+}
+.book {
+  position: absolute;
```
Comment (almuly) [inline styles/style.css:52]: remove this method and your page will ha a more readable position

File: index.html [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <title>Title</title>
+  </head>
+  <body class="body">
+    <div class="h1">
+      <h1>What do we offer</h1>
```
Comment (almuly) [inline index.html:11]: you can add tag hr <hr> with necessary styles for the bottom line

File: index.html [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <title>Title</title>
+  </head>
+  <body class="body">
+    <div class="h1">
```
Comment (almuly) [inline index.html]: Try to use styles name that does not cross with names of HTML attributes. In this case, for example, better to use block__title

**Comment by almuly** [comment]
Good job!  Fix remarks and for this task, it will be enough.

File: styles/style.css (hunk 1 above)
Comment (SobolevD51) [reply styles/style.css:52]: If I delete position method every element of my page is moving out of their place, I don't quite understand how to implement it

Comment (almuly) [reply styles/style.css:52]: Sorry. I was confused with the number of the string. You can remove position absolute from second string

File: styles/style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
+.body {
+  position: absolute;
```
Comment (almuly) [inline styles/style.css]: Remove this)

Comment (SobolevD51) [reply styles/style.css]: complite:)

**Pull request closed without merging**